docs/configs/site.js

```diff
@@ -45,7 +45,7 @@ export default {
         children: [
           {
             key: 'docs0',
-            text: 'latest(3.0.0)',
+            text: 'latest(3.1.0)',
```

Comment:
we should merge #12139 and show version 3.0.1 and 3.1.0 both in our website

Reply:
And there is one more thing here, if we merge this patch, the website will show the version 3.1.0 document, but we have three days for the ASF vote process.
But other change should be before the voting process, so our doc show we should separate into two PR instead of a single one

Reply:
> And there is one more thing here, if we merge this patch, the website will show the version 3.1.0 document, but we have three days for the ASF vote process.
> But other change should be before the voting process, so our doc show we should separate into two PR instead of a single one

I find out that your target branch is `3.1.0-prepare`, but this change should be in dev branch 
<img width="878" alt="image" src="https://user-images.githubusercontent.com/15820530/192230537-57db9e10-474f-4bc3-9533-49ac633bf4b3.png">, according to https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/contribute/release/release.html

Reply:
Have restored